src/main/java/co/inventorsoft/academy/dao/intetface/MatchDAOInterface.java

```diff
@@ -0,0 +1,46 @@
+package co.inventorsoft.academy.dao.intetface;
+
+import co.inventorsoft.academy.model.Match;
+
+import java.util.List;
+
+public interface MatchDAOInterface {
+    /**
+     * This method add Match to database
```

Comment:
```suggestion
     * This method add {@link Match} to database
```

This way you will see a class reference in Javadoc.